python/tank/commands/validate_config.py

```diff
@@ -84,6 +84,8 @@ def _run(self, log, parameters):
     
         # validate environments
         for env_name in parameters["envs"]:
+            log.info("")
+            log.info("Processing environment %s.yml" % env_name)
```

Comment:
Suggest leaving off the `.yml` and adding another log message after retrieving the `env` object. Then you can use `env.disk_location` to log the full path.
```python
log_info("Environment path: %s" % (env_name, env.disk_location)
```
Something like that. Maybe do the same in `cache_apps` as well?